engine/include/cubos/engine/font/atlas/glyph.hpp

```diff
@@ -0,0 +1,30 @@
+/// @file
+/// @brief Struct @ref cubos::engine::Glyph.
+/// @ingroup font-plugin
+
+#pragma once
+
+#include <glm/glm.hpp>
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Struct that holds glyph data such as texcoords, position offsets and advance.
+    ///
+    /// @ingroup font-plugin
+    struct CUBOS_ENGINE_API Glyph
```

Comment:
I would maybe rename this to `FontGlyph`, to avoid naming collisions. The file can keep the same name though :relieved: